.eslint/__tests__/svg-needs-viewBox.test.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * @fileoverview All svgs should have a viewBox attribute
+ * @author Glorilí Alejandro
+ */
+'use strict';
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require('../rules/svg-needs-viewBox.js');
+
+const RuleTester = require('eslint').RuleTester;
+
+RuleTester.setDefaultConfig({
+  parserOptions: {
+    ecmaVersion: 6,
+    ecmaFeatures: {
+      jsx: true,
+    },
+  },
+});
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+var ruleTester = new RuleTester();
```

Comment:
question: is there a reason this uses `var` instead of `const`?

Reply:
ah this was left in from a source, good catch! Just changed it.